archivers/libzzip/files/patch_manPython.diff

```diff
@@ -0,0 +1,87 @@
+diff --git docs/CMakeLists.txt.orig docs/CMakeLists.txt
+index 9271dd9..62795cf 100644
+--- docs/CMakeLists.txt.orig
++++ docs/CMakeLists.txt
+@@ -11,6 +11,7 @@ option(BUILD_TESTS "Build test programs" OFF)
+ option(MSVC_STATIC_RUNTIME "Build with static runtime libs (/MT)" ON)
+ option(ZZIP_HTMLSITE "Generate site html pages from docs" OFF)
+ option(ZZIP_HTMPAGES "Generate html manpages from sources" OFF)
++set(PYTHON_EXECUTABLE "" CACHE FILEPATH "Path to Python executable")
+ 
+ if(UNIX)
+ option(ZZIP_MANPAGES "Generate man3 manpages from sources" ON)
+@@ -21,7 +22,6 @@ endif()
+ # Zlib library needed
+ find_package ( ZLIB REQUIRED )
+ # pkg_search_module ( ZZIP zzip )
+-find_package(PythonInterp 3.5 REQUIRED)
+ find_package(UnixCommands REQUIRED) # bash cp mv rm gzip tar
+ find_program(XMLTO xmlto)
+ 
+@@ -31,7 +31,6 @@ set(README ${CMAKE_SOURCE_DIR}/README)
+ set(topdir ${CMAKE_SOURCE_DIR})
+ set(srcdir ${CMAKE_CURRENT_SOURCE_DIR})
+ set(outdir ${CMAKE_CURRENT_BINARY_DIR})
+-set(PY ${PYTHON_EXECUTABLE})
+ 
+ set(mandir ${CMAKE_INSTALL_FULL_MANDIR})
+ set(docdir ${CMAKE_INSTALL_FULL_DOCDIR})
+@@ -80,7 +79,7 @@ add_custom_command(OUTPUT site.html
+ add_custom_target(site DEPENDS site.html)
+ 
+ add_custom_command(OUTPUT zzip.xml
+-    COMMAND ${PY} ${srcdir}/zzipdoc/htm2dbk.py ${htm_FILES} zziplib.xml -o ${outdir}/zzip.xml
++    COMMAND ${PYTHON_EXECUTABLE} ${srcdir}/zzipdoc/htm2dbk.py ${htm_FILES} zziplib.xml -o ${outdir}/zzip.xml
```

Comment:
What if we defined `PY` instead of `PYTHON_EXECUTABLE` instead? Then only the above two lines would be needed, while the rest could be left intact?